src/pages/SearchPage/SearchPage.tsx

```
@@ -0,0 +1,120 @@
+        </div>
+
+        {/* Filters Panel */}
+        {showFilters && (
+          <div className="rounded-lg bg-white p-4 shadow-sm">
+            <h3 className="mb-4 text-h4 font-semibold">Filters</h3>
+            <div className="grid grid-cols-1 gap-4 md:grid-cols-3">
+              {/* TODO: Add filter components */}
+              <div>
+                <label className="text-bodySmall mb-2 block font-medium">
+                  Category
+                </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)